qiskit/bell_states_qiskit.py

```diff
@@ -3,8 +3,7 @@
 from argparse import RawTextHelpFormatter
 
 from iqm.qiskit_iqm import IQMProvider
-
-from qiskit import Aer, QuantumCircuit, QuantumRegister, execute
+from qiskit_aer import Aer, QuantumCircuit, QuantumRegister, execute
```

Comment:
`from qiskit import QuantumCircuit, QuantumRegister, execute`
`from qiskit_aer import Aer`

Also, I think `execute` should not be used anymore

Reply:
Can we just update course material on this pull request. These old examples have lot more problems